lib/Doctrine/Common/DataFixtures/Loader.php

```diff
@@ -322,6 +322,10 @@ private function getUnsequencedClasses($sequences, $classes = null)
         }
 
         foreach ($classes as $class) {
+            if(!isset($sequences[$class])) {
+                dump('UNDEFINED INDEX');
+                die;
```

Comment:
Unfortunately, without this check the notice is suppressed and the test passes. I am not sure why. This is rather demo than proper PR.